quizapp/src/test/groovy/billennium/quizapp/QuizSpec.groovy

```diff
@@ -0,0 +1,118 @@
+package billennium.quizapp
+
+import billennium.quizapp.controller.QuizController
+import billennium.quizapp.entity.*
+import billennium.quizapp.repository.*
+import billennium.quizapp.resource.AnswerDto
+import billennium.quizapp.resource.QuestionDto
+import billennium.quizapp.resource.QuizDefinitionDto
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.http.MediaType
+import org.springframework.test.web.servlet.MockMvc
+import spock.lang.Specification
+
+import static billennium.quizapp.controller.ControllerConstants.*
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class QuizSpec extends Specification {
+
+    @Autowired
+    private MockMvc mockMvc
+    @Autowired
+    private QuizController quizController
+
+    @Autowired
+    private CandidateRepository candidateRepository
+
+    @Autowired
+    private AnswerRepository answerRepository
+
+    @Autowired
+    QuestionRepository questionRepository;
+
+    @Autowired
+    private QuizDefinitionRepository quizDefinitionRepository
+
+    @Autowired
+    private QuizExecutedRepository quizExecutedRepository
+
+    def candidate
+
+    def model;
+
+    def setup() {
```

Comment:
może idzie uprościć inicjalizacje danych ? W DataLoader jest zrobione że wszystko zapisuje się odwołaniem do jednego repozytorium bo kaskadować ustawiona jest na zapis.

Reply:
okej postaram sie to uproscic